packages/elements/src/tree/managers/tree-manager.ts

```diff
@@ -430,23 +433,23 @@ export class TreeManager<T extends TreeDataItem> {
   }
 
   /**
-   * Expands all items in the tree
+   * Expands all items managed by the Tree Manager.
```

Comment:
Why it's necessary to mention 'managed by the Tree Manager'? I'd assume that there are possible that some items are not managed by Tree Manager and those items won't be collapsed. Is that right?

Reply:
There is no such case: all items must be managed by Tree Manager.  Is `Expands all items` too short?

Reply:
Looks OK to me. Concise and clear.